ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -81,7 +81,7 @@ Set Server Size
     ...    Can be "Small", "Medium", "Large", "Custom"
     ...    If "Custom", need CPU request/limit and Memory request/limit
     [Arguments]    ${size}
-    Open Model server size Options Menu  # robocop:disable
+    Open Model server size Options Menu
```

Comment:
Can you change it to use camel case?

Reply:
The case is based on the menu name in roads since the keyword asks for the menu name. 
<img width="1056" alt="Screenshot 2023-11-02 at 11 18 41" src="https://github.com/red-hat-data-services/ods-ci/assets/11557840/f8fd7f89-713a-48ec-bd2f-952464dedf11">

Reply:
ah it is `Open ${section} Options Menu`. Then could you add the `  # robocop:disable` comment back? otherwise it will complain

Reply:
Sure